build/runtimes.json

```diff
@@ -0,0 +1,3774 @@
+﻿// RID catalog file distributed in the Microsoft.NETCore.Platforms package and maintained here:
+// https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json
+// This information is primarily used in self-contained scenarios when the RID graph is not available
+// at runtime.
+
+{
+  "runtimes": {
```

Comment:
Reference file in Host - [Link](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/runtimes.json)